setup.cfg

```diff
@@ -134,7 +134,7 @@ no_implicit_optional=True
 ignore_errors=True
 
 [mypy-pandas.tests.api.test_api]
-ignore_errors=True
+ignore_errors=False
```

Comment:
can just remove entirely as the default for ignore_errors is False

Reply:
do i need to remove entire section
 - [mypy-pandas.tests.api.test_api]
 - ignore_errors=True

Reply:
yes

Reply:
done

Reply:
remove L134-136